packages/client/utils/mediaRoom/Logger.ts

```diff
@@ -0,0 +1,21 @@
+import debug from 'debug'
```

Comment:
may be worthwhile (or at least fun to try) to see if you can only import `debug` if we're in development. that way in prod it won't get bundled.

Reply:
Hmm I want to be able to switch on debug logs on a development server environment. So far it's been really helpful to surface bugs that only come up in a deployed environment. Is there any env var that is specific only to production? Or maybe I can try to import it only if `DEBUG=mediasoup*` env var is set...

Reply:
`LoggerFactory` now instantiates `NoOpLogger` or `DebugLogger` depending on value of `process.env.DEBUG`. `debug` module is dynamically imported in `DebugLogger` creation.